contentsquare/src/main/java/com/tealium/remotecommands/contentsquare/ContentsquareInstance.kt

```diff
@@ -42,6 +44,33 @@ class ContentsquareInstance(private val application: Application) : Contentsquar
         }
     }
 
+    override fun sendUserIdentifier(userId: String) {
+        Log.d(TAG, "Sending user identifier: $userId")
+        Contentsquare.sendUserIdentifier(userId)
+    }
+
+    override fun sendCustomVars(screenName: String, customVars: Array<JSONObject>) {
+        Log.d(TAG, "Sending custom variables for screen: $screenName")
+        val csCustomVars = customVars.mapNotNull { json ->
+            val index = json.optInt(CustomVars.INDEX, -1)
+            val name = json.optString(CustomVars.NAME)
+            val value = json.optString(CustomVars.VALUE)
+
+            if (index in 1..20) {
+                CustomVar(index, name, value)
+            } else {
+                Log.e(TAG, "Invalid custom var index: $index (must be 1..20). Skipping: $json")
+                null
+            }
+        }.toTypedArray()
+        
+        if (csCustomVars.isNotEmpty()) {
+            Contentsquare.send(screenName, csCustomVars)
+        } else {
+            Log.d(TAG, "No valid custom vars to send")
+        }
+    }
```

Comment:
think this was already raised in the iOS PR, but this command should probably just be combined with the regular `send` command for screenviews, since they are related.
Make the `customVars` array optional, and use that to decide which method to call.